<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineScale GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/linescaleGUI/logo/logo.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widgetChart" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutReadingsRight">
      <item>
       <widget class="QGroupBox" name="groupCurrent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Current force</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lblCurrentForce">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 48pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>0.00 kN</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSetZero">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Set zero</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupPeak">
        <property name="title">
         <string>Peak force</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblPeakForce">
           <property name="styleSheet">
            <string notr="true">font: 700 48pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>0.00 kN</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnResetPeak">
           <property name="text">
            <string>Reset peak value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionGitHub"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfigure"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionStartStop"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../../assets/resource.qrc">
     <normaloff>:/linescaleGUI/icons/connect.png</normaloff>:/linescaleGUI/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="toolTip">
    <string>Connect to device</string>
   </property>
  </action>
  <action name="actionStartStop">
   <property name="icon">
    <iconset resource="../../assets/resource.qrc">
     <normaloff>:/linescaleGUI/icons/start.png</normaloff>:/linescaleGUI/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>StartStop</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../../assets/resource.qrc">
     <normaloff>:/linescaleGUI/icons/disconnect.png</normaloff>:/linescaleGUI/icons/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from device</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../assets/resource.qrc"/>
 </resources>
 <connections/>
</ui>
